wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -1026,6 +1040,11 @@ private void broadcastPeerState(final int numPeers) {
     }
 
     private void handleBlockchainStateNotification(BlockchainState blockchainState) {
+        // send this out for the Network Monitor, other activities observe the database
+        final Intent broadcast = new Intent(ACTION_BLOCKCHAIN_STATE);
+        broadcast.setPackage(getPackageName());
+        LocalBroadcastManager.getInstance(this).sendBroadcast(broadcast);
```

Comment:
There may be a better way to do this in the BlockListFragment, with a database observer.  For now this is the quickest way to trigger the block loader in BlockListFragment with out more significant rewrites.